### PR TITLE
Use user-provided app name to install app

### PR DESCRIPTION
**Pull request by ArturPimentel**
#### What is the purpose of this pull request?
To use user-provided app name to install app

#### What problem is this solving?
Trying to install an app by specifying its version would not give the expected result. This was caused by the use of the last published manifest to create the appId used to install the app. Now, all the interpretation of the user-provided app name is given to Apps.

#### How should this be manually tested?
1. Clone/pull this repo/branch and link in your workspace
2. Install any app by specifying its version, for example:
```
vtex install vtex.sms-provider@0.4.1
```
3. The version installed should be the one you specified

#### Screenshots or example usage
None to show

#### Types of changes
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

File: node/graphql/resolvers/install.ts [hunk 1]
```diff
@@ -11,10 +11,11 @@ const buildInstallResponse = (code: InstallStatus, appManifest: any = null): Ins
   }
 }
 
-const installAndLog = async (ctx: ColossusContext, appId: string, registry: string, installStatus: InstallStatus, contract: any) => {
+const installAndLog = async (ctx: ColossusContext, appName: string, registry: string, installStatus: InstallStatus, contract: any) => {
   const { resources: {apps, logger}, vtex: { account } } = ctx
   const free = !contract
-  await apps.installApp(appId, registry)
```
Comment (marcelotokarnia) [inline node/graphql/resolvers/install.ts]: apps.installApp should receive some information about the app major ... should it not ?

File: node/graphql/utils/index.ts [hunk 2]
```diff
@@ -34,3 +36,17 @@ export const buildPolicyPayload = (appManifest: AppManifest) => {
...
+}
+
+export const getMajor = (appName: string): string => {
+  const version = head(tail(split('@', appName)))
+  if (version) {
+    const major = head(split('.', version))
+    return major
+  }
+  return 'x'
+}
```
Comment (marcelotokarnia) [inline node/graphql/utils/index.ts]: 😂

Comment (ArturPimentel) [reply node/graphql/utils/index.ts]: 😭

File: node/graphql/resolvers/install.ts (hunk 1 above)
Comment (ArturPimentel) [reply node/graphql/resolvers/install.ts]: The proper major is inferred beforehand

**Pull request merged**